cmd/clusterawsadm/cmd/gc/disable.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Missing unit test for this folder. It would be nice to add some trivial checks.

Reply:
None of the other cobra command functions have tests. Lets follow-up on this in the future.